<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/icons/browser.png</normaloff>:/icons/icons/browser.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>160</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>20</number>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>0</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Array A1</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>53</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>83</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>78</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>36</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>95</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>46</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>98</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>45</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_max">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ave">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string>Average</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sort_to_higher">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string>Sort to larger</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/signal-alt-2.png</normaloff>:/icons/icons/signal-alt-2.png</iconset>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>19</y>
      <width>70</width>
      <height>141</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="movement">
     <enum>QListView::Static</enum>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>8</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.7</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_input_table_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_output_table_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/list.png</normaloff>:/icons/icons/list.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_iput_list_widget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_output_list_widget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/list.png</normaloff>:/icons/icons/list.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_array">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>500</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sort_lower">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string>Sort to lower</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/signal-alt-2-zercalled.png</normaloff>:/icons/icons/signal-alt-2-zercalled.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_N">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;N&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Result_name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Result">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult_n">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string>Array*N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/plus.png</normaloff>:/icons/icons/plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/minus.png</normaloff>:/icons/icons/minus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n_element">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string>Array[n]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Array A2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_n_element_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
    </property>
    <property name="text">
     <string>Set array size n</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
